### PR TITLE
[desktop] Add first use buttons to main area

### PR DESCRIPTION
**Pull request by twiss**
This adds some buttons to the main (welcome) area on the desktop to help a first-time user do something useful with the app as I talked about in https://github.com/codexa/firetext/issues/131#issuecomment-72144915.

Adds one l10n string.

### CODE REVIEW

File: locales/en-US/app.properties [hunk 1]
```diff
@@ -26,6 +26,7 @@ documents=Documents
 documents-device=Documents on this Device
 documents-dropbox=Documents on your Dropbox
 done=Done
+dropbox-connect=Connect to Dropbox
```
Comment (ferndot) [inline locales/en-US/app.properties]: I don't think we should add another string right now. Can we use `use-dropbox` instead? (And we can change that string later).

**Comment by ferndot** [comment]
This is a really nice feature :+1: !

**Comment by twiss** [comment]
Rebased and changed to use `use-dropbox`.

**Comment by ferndot** [comment]
Great, thanks!

**Pull request merged**